examples/toolkit/scripts/compare.py

```
@@ -0,0 +1,90 @@
+            "engagement_ratio": round(
+                s1["total_score"] / max(s2["total_score"], 1), 2
+            ),
+        }
+
+        # Remove author lists from per-sub stats (redundant)
+        del result[sub1]["authors"]
+        del result[sub2]["authors"]
+
+        json.dump(result, sys.stdout, indent=2)
+        print()
+    except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)